### PR TITLE
:sparkles: feat(root): adad

### PR DESCRIPTION
**Pull request by gauravCodeBuddy**
ad

BREAKING CHANGE:
ada

ISSUES CLOSED: dada

### CODE REVIEW

File: apps/admin/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+
```
Comment (gauravCodeBuddy) [inline apps/admin/README.md:4]: This is okay

File: apps/admin/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
+- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+
+## Expanding the ESLint configuration
+
+If you are developing a production application, we recommend updating the configuration to enable type aware lint rules:
+
+- Configure the top-level `parserOptions` property like this:
+
```
Comment (gauravCodeBuddy) [inline apps/admin/README.md:15]: THis not good

File: apps/admin/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```js
```
Comment (gauravCodeBuddy) [inline apps/admin/README.md:16]: This is oaky

File: apps/admin/README.md (hunk 2 above)
Comment (gauravCodeBuddy) [reply apps/admin/README.md:15]: okay

File: apps/admin/README.md (hunk 1 above)
Comment (gauravkjha123) [reply apps/admin/README.md:4]: got a point

File: apps/admin/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+export default {
+  // other rules...
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    project: ['./tsconfig.json', './tsconfig.node.json'],
```
Comment (gauravkjha123) [inline apps/admin/README.md:22]: this is review 1

Comment (gauravCodeBuddy) [reply apps/admin/README.md:22]: this is coment 1

Comment (gauravCodeBuddy) [reply apps/admin/README.md:22]: > this is coment 1

jnj

**Pull request closed without merging**